chainio/clients/builder.go

```diff
@@ -180,7 +218,7 @@ func (config *BuildAllConfig) buildElClients(
 		txMgr,
 	)
 
-	return elChainReader, elChainWriter, nil
+	return elChainReader, elChainWriter, avsRegistryContractBindings, elContractBindings, nil
```

Comment:
wait why is this retuning `avsRegistryContractBindings` even tho it's not returning any AVS clients?

Reply:
true this shouldn't be here

Reply:
Okie, i am adjusting my PR.

Reply:
lgtm now, thanks